broker/consumer/ps-connector.properties

```diff
@@ -1,7 +1,8 @@
+# reference: https://docs.confluent.io/platform/current/installation/configuration/connect/sink-connect-configs.html
 name=ps-sink-connector
 connector.class=com.google.pubsub.kafka.sink.CloudPubSubSinkConnector
 tasks.max=10
-# set kafka the topic
+# set kafka the topic(s)
 topics=KAFKA_TOPIC
```

Comment:
@hernandezc1 Here, `KAFKA_TOPIC` can be a comma-separated list of topics (not a python list -- no brackets, just commas). I added the reference link in a code comment at the top of the file. When you're ready to connect to multiple topics, try setting the `KAFKA_TOPIC_FORCE` environment variable on the VM to the appropriate list of topics before turning it on. Let me know how it goes. This will be highly preferably to running multiple VMs, assuming it works.